### ISSUE TITLE
Automatic installation of the plugin failed since few hours

### ISSUE THREAD

**Issue by Roms63** [phonegap-build/PushPlugin#603]
Hello,

since today, the automatic installation of the plugin (thanks to xdk/cordova) does not work anymore. The building of a simple project with nothing, except https://github.com/phonegap-build/PushPlugin.git as an external plugin failed. (may be a problem since the last update of the plugin....)

Regards,
Romain.

**Comment by langowskiadam** [comment]
I've got the same problem. Building app in intelXDK is failing since todey. When I remove PushPlugin from plugin list build is successful

**Comment by Roms63** [comment]
Yep. I have found a "workaround" if it can help you. You can download the release 2.4.0 directly in your project and installed it as a local plugin. It works fine. So the problem is really with the current version.

**Comment by jparraguez** [comment]
Hi everyone, i have the same problem, please teach me for use the manually install i not good for this and my english is so bad. thanks

**Comment by jparraguez** [comment]
oo sorry i have intel xdk 2248.

**Comment by Roms63** [comment]
Hi.

don't worry, it is very simple. You just have to download the last archive of the plugin from the github website, uncompress it, and copy the directory of the plugin in your xdk project (example : your_project_name/www/PushPlugin).

After that, the process is exactly the same than for an automatic installation, except you will choose in the "third party plugin" the option "import local plugin" vs "Get Plugin from the web", and you will indicate the local directory of the plugin on your computer drive vs the "http..github...." address of the plugin.

**Comment by MichaelO23** [comment]
Hi. I still get the same error with the plugin imported locally.

The build failed. An error occurred while building the application. Verify your build assets are correct and try again.
Build Log:
Building a Cordova 4.1.2 application.
The application name is "e-Habillement App"
The package name is "com.sca.ehab"
Plugin "org.apache.cordova.device" (0.2.13) installed.
Plugin "org.apache.cordova.splashscreen" (0.3.5) installed.

**Comment by Roms63** [comment]
Have you download this version : https://github.com/phonegap-build/PushPlugin/releases/tag/2.4.0 ?

**Comment by MichaelO23** [comment]
Thanks, it's working for the Android and iOS builds but not for Windows 8

**Comment by Roms63** [comment]
You are welcome ! sorry, may be this PushPlugin version is not compatible with Windows 8... I developped an application only for IOS and Android ;)  This solution is just a "workaround", I hope a better understanding about what is the problem and a fix will be found, but I am not strong enough for that.

**Comment by jparraguez** [comment]
Roms63 thanks you very much :D is work know on local, you know how test the push on ios? i have the token id but i can't test, if you know please help me.

PD: I have developer ios account active, but i dont have a apple mac.

**Comment by Roms63** [comment]
You are welcome... for testing on IOS, I am in the same situation than you. I will start today my tests on IOS, and it will be my first time. All my tests until today are done on android. I will start with that : http://qnimate.com/building-intel-xdk-app-for-adhoc-and-app-store/

I will keep you inform.
